src/lando/main/models/repo.py

```diff
@@ -47,23 +44,55 @@ class RepoError(Exception):
     pass
 
 
-# Dict of hook names to docstring, suitable as human-friendly choices.
-HOOKS_CHOICES = {chk.__name__: chk.__doc__ or chk.__name__ for chk in ALL_CHECKS}
-
-
 def get_default_hooks() -> list[str]:
     """Returns a list of all known hook names, suitable as a default value.
 
     Assuming a normal repository, we enable everything but the BugReferencesCheck, which
     is only relevant for Try-type repos."""
-    return [hook for hook in HOOKS_CHOICES if hook != BugReferencesCheck]
+    return [
+        hook.name
+        for hook in Repo.HooksChoices
+        if hook.name != BugReferencesCheck.name()
+    ]
 
 
 class Repo(BaseModel):
     """Represents the configuration of a particular repo."""
 
     _scm: AbstractSCM | None = None
 
+    class HooksChoices(models.TextChoices):
+        """Allowable statuses of a tree."""
```

Comment:
Can this be moved to the top-level? Also, the docstring looks copied from the `TreeStatus` class.